tigerpath/majors_and_certificates/majors/NEU_2018.json

```
@@ -0,0 +1,348 @@
+                                "ISC 231",
+                                "ISC 232",
+                                "ISC 233",
+                                "ISC 234"
+                            ]
+                        }
+                    ]
+                },
+                {
+                    "name": "Electives",
+                    "max_counted": 1,
+                    "min_needed": 5,
+                    "explanation": "Students must take 5 NEU courses from 3 out of 4 subject areas.",
+                    "completed_by_semester": 8,
+                    "req_list": [
+                        {
+                            "name": "Molecular/ Cellular/ Disease",
+                            "max_counted": 5,
+                            "min_needed": null,
+                            "explanation": "",
+                            "completed_by_semester": 8,
+                            "course_list": [
+                                "NEU 301: Cellular Neurobiology",
+                                "EEB/NEU 403: Genes and Neurons Underlying Behavioral Evolution",
+                                "NEU/PSY 413: The Neurobiology of Stress",
+                                "MOL/NEU 447: Neuroimmunology",
+                                "MOL/NEU 451: Genes, Brains, Minds",
+                                "MOL 459: Basic Virology",
+                                "MOL 458: Psychopharmacology",
+                                "NEU 475: Basal Ganglia in Health and Disease",
+                                "NEU 501A: Molecules to Systems"
+                            ]
+                        },
+                        {
+                            "name": "Neural Computation",
+                            "max_counted": 5,
+                            "min_needed": null,
+                            "explanation": "",
+                            "completed_by_semester": 8,
+                            "course_list": [
+                                "PSY/NEU 330: Connectionist Modeling",
+                                "PSY/NEU 338: Animal Learning & Decision-making",
+                                "PSY/NEU 340: Neuroeconomics",
+                                "NEU 437: Computational Neuroscience",
+                                "NEU 457: Measurement and Analysis of Neural Circuit Dynamics",
+                                "COS 402: Artificial Intelligence",
+                                "COS 324: Fundamentals of Machine Learning",
+                                "COS 485: Neural Networks: Theory and Applications",
+                                "ELE 364: Machine Learning for Predictive Data Analytics",
+                                "MAT/APC 323: Topics in Mathematical Modeling - Mathematical Neuroscience"
+                            ]
+                        },
+                        {
+                            "name": "Systems & Circuits",
+                            "max_counted": 5,

```

Maybe this should be a 3, so that they're forced to take the other two courses from another subject area?
```suggestion
                            "max_counted": 3,
```